common/src/main/java/com/wynntils/features/utilities/InputTranscriptionFeature.java

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.utilities;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.config.RegisterConfig;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.mc.event.ChatScreenKeyTypedEvent;
+import com.wynntils.mc.event.ChatSentEvent;
+import com.wynntils.mc.event.CommandSentEvent;
+import com.wynntils.mc.event.EditBoxInsertEvent;
+import com.wynntils.mc.event.ScreenInitEvent;
+import com.wynntils.models.wynnalphabet.WynnAlphabet;
+import com.wynntils.screens.transcription.widgets.WynnAlphabetButton;
+import com.wynntils.utils.mc.McUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.client.gui.components.EditBox;
+import net.minecraft.client.gui.screens.ChatScreen;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
+
+@ConfigCategory(Category.CHAT)
+public class InputTranscriptionFeature extends Feature {
+    @RegisterConfig
+    public final Config<Boolean> useBrackets = new Config<>(false);
+
+    private static final int MAX_CHAT_LENGTH = 256;
+    // Numbers higher than this will be replaced with "∞"
+    private static final int MAX_TRANSCRIPTABLE_NUMBER = 5000;
+    private static final Pattern NUMBER_PATTERN = Pattern.compile("\\d+");
+
+    @SubscribeEvent
+    public void onScreenInit(ScreenInitEvent event) {
+        if (useBrackets.get()) return;
+
+        if (event.getScreen() instanceof ChatScreen chatScreen) {
```

Comment:
Maybe make a config for the buttons, in case you only want to use brackets?

Reply:
That's how it currently works. by default brackets will do nothing and you have to use the buttons, toggling the brackets on will remove the buttons and transcribe what you put in brackets

Reply:
Unless you think always transcribing what's in brackets regardless of if the buttons are enabled or not is a good idea

Reply:
Works quite nicely if you nest them too.
![image](https://github.com/Wynntils/Artemis/assets/8337467/9d39a657-1566-4241-8509-80ec879fbf13)